src/cli/minimist.js

```diff
@@ -1,14 +1,14 @@
-"use strict";
+'use strict';
 
-const minimist = require("minimist");
+const minimist = require('minimist');
 
 const PLACEHOLDER = null;
 
 /**
  * unspecified boolean flag without default value is parsed as `undefined` instead of `false`
  */
 module.exports = function(args, options) {
-  const boolean = options.boolean || [];
+  const { boolean } = options || [];
```

Comment:
This should be `const { boolean = [] } = options`.

Reply:
Yes ... thanks... i will update that... i missed it...